.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will generate a release distribution and upload it to PyPI
+
+name: Publish Build and GitHub Release
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  tag_release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get Version
+        run: |
+          VERSION=$(python setup.py --version)
+          echo "VERSION=${VERSION}" >> $GITHUB_ENV
+      - uses: ncipollo/release-action@v1
+        with:
+          token: ${{secrets.GITHUB_TOKEN}}
+          tag: ${{env.VERSION}}
+  build_and_publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install Build Tools
+        run: |
+          python -m pip install build wheel
+
+      - name: Build Distribution Packages
+        run: |
+          python setup.py bdist_wheel
+      - name: Publish to Test PyPI
```

Comment:
I don't see a `setup.py` file; is this pip installable? If not, the version tagging is worth keeping, but the `build_and_publish` action could be removed

Reply:
> I don't see a `setup.py` file; is this pip installable? If not, the version tagging is worth keeping, but the `build_and_publish` action could be removed

I has setup.py in root

Reply:
Can you add build tests to unit tests (or another workflow)? If they aren't passing, PyPI publication can be omitted here and left for another PR. This workflow would close #20
```yaml
  build_tests:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: Setup Python
        uses: actions/setup-python@v1
        with:
          python-version: 3.8
      - name: Install Build Tools
        run: |
          python -m pip install build wheel
      - name: Build Distribution Packages
        run: |
          python setup.py bdist_wheel
```